specificatie/genereervariant/openapi.json

```diff
@@ -4338,6 +4338,13 @@
             "items" : {
               "$ref" : "#/components/schemas/HalLink"
             }
+          },
+          "bijbehorendeErfpachters" : {
+            "type" : "array",
+            "description" : "Zakelijk gerechtigden die erfpachter zijn, omdat hun zakelijk recht is ontstaan uit een grondperceel belast met erfpacht. Dit zijn de erfpachters van het perceel OF de eigenaren van een appartementsrecht op het perceel.\n",
```

Comment:
Toch nog een wijziging n.a.v. de review van Thijs Pothoven:
description:
Zakelijk gerechtigden die erfpachter zijn, omdat zij eigenaar zijn van een recht van erfpacht OF hun zakelijk recht is ontstaan uit een splitsing van het recht van erfpacht. Dit zijn de erfpachters van het perceel OF de eigenaren van een appartementsrecht dat is ontstaan uit de splitsing van het recht van erfpacht van het perceel